web-console/src/variables.ts

```diff
@@ -21,10 +21,10 @@ export const LEGACY_OVERLORD_CONSOLE = '/console.html';
 
 export const DRUID_WEBSITE = 'http://druid.io';
```

Comment:
This one also?

Reply:
Nice, travis caught this one https://travis-ci.org/apache/incubator-druid/jobs/547030885#L1745